src/main/java/core/basesyntax/service/FruitTransaction.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.service;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int amount;
```

Comment:
The encapsulation principle is violated here. All fields of the class must be private, and access to them should be only through getters and setters.